src/tests/ftest/pool/pool_svc.py

```diff
@@ -43,64 +45,63 @@ def test_poolsvc(self):
+        self.pool = TestPool(self.context, dmg_command=self.get_dmg_command())
+        self.pool.get_params(self)
+        self.pool.svcn.update(createsvc[0])
         try:
-            # initialize a python pool object then create the underlying
-            # daos storage
-            self.pool = DaosPool(self.context)
-            self.pool.create(createmode, createuid, creategid,
-                             createsize, createsetid, None, None, createsvc[0])
-            self.pool.connect(1 << 1)
-
-            # checking returned rank list for server more than 1
-            iterator = 0
-            while (
-                    int(self.pool.svc.rl_ranks[iterator]) > 0 and
-                    int(self.pool.svc.rl_ranks[iterator]) <= createsvc[0] and
-                    int(self.pool.svc.rl_ranks[iterator]) != 999999
-            ):
-                iterator += 1
-            if iterator != createsvc[0]:
-                self.fail("Length of Returned Rank list is not equal to "
-                          "the number of Pool Service members.\n")
-            rank_list = []
-            for iterator in range(createsvc[0]):
-                rank_list.append(int(self.pool.svc.rl_ranks[iterator]))
-                if len(rank_list) != len(set(rank_list)):
-                    self.fail("Duplicate values in returned rank list")
-
-            self.pool.pool_query()
-            leader = self.pool.pool_info.pi_leader
-            if createsvc[0] == 3:
-                # kill pool leader and exclude it
-                self.pool.pool_svc_stop()
-                self.pool.exclude([leader])
-                # perform pool disconnect, try connect again and disconnect
-                self.pool.disconnect()
-                self.pool.connect(1 << 1)
-                self.pool.disconnect()
-                # kill another server which is not a leader and exclude it
-                server = DaosServer(self.context, self.server_group, 3)
-                server.kill(1)
-                self.pool.exclude([3])
-                # perform pool connect
-                self.pool.connect(1 << 1)
-
-            if expected_result in ['FAIL']:
-                self.fail("Test was expected to fail but it passed.\n")
-
-        except DaosApiError as excep:
+            self.pool.create()
+            if expected_result == RESULT_FAIL:
+                self.fail("Test was expected to fail, but it passed.\n")
+        except TestFail as excep:
+            print("## TestFail exception is caught at pool create!")
             print(excep)
             print(traceback.format_exc())
-            if expected_result == 'PASS':
+            if expected_result == RESULT_PASS:
+                self.fail("Test was expected to pass but it failed.\n")
+
+        # FAIL case should fail at above pool create, so do below only for
+        # PASS case
+        if expected_result == RESULT_PASS:
+            self.log.debug("self.pool.svc_ranks = %s" % self.pool.svc_ranks)
```

Comment:
(pylint-logging-not-lazy) Specify string format arguments as logging function parameters